src/components/CheckBox.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const CheckBox = ({onChange}) => {
+    const onCheckboxChangeHandler = (event) => {
+        let isChecked = event.target.checked;
+        onChange(isChecked);
+    }
+
+    return (
+        <div className ="checkbox-container">
+            <input type="checkbox" id="checkbox" className="checkbox" onChange={onCheckboxChangeHandler}/>
```

Comment:
make it controlled 
read it to understand why u need it https://stackoverflow.com/questions/42522515/what-are-react-controlled-components-and-uncontrolled-components#:~:text=A%20Controlled%20Component%20is%20one,props%20to%20the%20controlled%20component.

Reply:
thx